src/actionCreators/resourceHelpers.js

```diff
@@ -195,8 +195,8 @@ const newProperty = (subject, propertyTemplate, noDefaults, errorKey) => (dispat
     property.show = true
   }
 
-  // If required, then expand the property.
-  if (propertyTemplate.required) {
+  // If required and we do not already have some default values, then expand the property.
+  if (propertyTemplate.required && !property.values) {
```

Comment:
this is the actual bug fix that prevents overwriting the default values when required (if default values happen to exist)